src/us/kbase/narrativejobservice/test/CondorIntegrationTest.java

```diff
@@ -373,16 +374,15 @@ public void testSimpleJobWithParent() throws Exception{
 
         List<String> subjobs = (List<String>)ret.getAdditionalProperties().get("sub_jobs");
         System.out.println("Asserting child jobs match sub jobs");
-       assertEquals(child_jobs,subjobs);
-
+        assertTrue(child_jobs.containsAll(subjobs) && subjobs.containsAll(child_jobs));
```

Comment:
Turns out this test was a bit flaky, and this fixes it by ignoring order of the two lists